web-ui/src/pages/ViewFeedbackPage.jsx

```diff
@@ -380,16 +409,17 @@ const ViewFeedbackPage = () => {
                 </InputAdornment>
               )
             }}
+            value={searchText}
           />
-          <FormControl className={classes.textField} value={dateRange}>
+          <FormControl className={classes.textField}>
             <TextField
               id="select-time"
               select
               fullWidth
               size="small"
               label="Show requests sent within"
               onChange={e => setDateRange(e.target.value)}
-              defaultValue={DateRange.THREE_MONTHS}
+              value={dateRange}
```

Comment:
We don't want to always default to THREE_MONTHS, but instead want to use the value in the corresponding query parameter.